locale/zh_CN.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -287,7 +287,7 @@ msgstr "在左上角，点击VNC Server"
 
 #: index.php:1226
 msgid "Select &quot;Reverse Connection&quot;."
-msgstr "选择撤销连接"
+msgstr "选择‘撤销连接’"
```

Comment:
'Reverse Connection' is text in another program (which has no Chinese translation), so it should not translated at all but cited unmodified.